admin/build-release.sh

```diff
@@ -54,6 +54,11 @@ if [ "X${GMT_DCW_SOURCE}" = "X" ]; then
 	exit 1
 fi
 
+if [ $(egrep -c '^set \(GMT_PUBLIC_RELEASE TRUE\)' cmake/ConfigDefault.cmake) -eq 0 ]; then
```

Comment:
We set GMT_PUBLIC_RELEASE to TRUE in `admin/ConfigReleaseBuild.cmake`. Why do we need to check it here?

Reply:
Forgot. Just removed it

Reply:
@PaulWessel I just realized that the check is still necessary. 

The build-release.sh script also builds the source tarballs. If we forget to set GMT_PUBLIC_RELEASE to TRUE, the macOS bundle is still correctly built, but the ConfigDefaults.cmake in the source tarballs have a FALSE GMT_PUBLIC_RELEASE.

Reply:
OK, will reinstate.